.github/workflows/main.yaml

```diff
@@ -221,3 +221,20 @@ jobs:
           fetch-depth: 0
 
       - run: make image
+
+  buildx:
```

Comment:
Seems good. Have 3 questions to ask.
1 Does this work if the users don't have push permission for the image?  Do we need a script for docker logining?
2 If this CI is too slow to build and may affect other CI actions, why not move it out? 
3 Does this CI need to be daily built?  If so, we need to add a schedule for it.